integreat_cms/xliff/utils.py

```diff
@@ -395,7 +395,7 @@ def xliff_import_confirm(request, xliff_dir):
                     # Update existing translation
                     existing_translation = page_translation.latest_version
                     existing_translation.currently_in_translation = False
-                    existing_translation.save()
+                    existing_translation.save(update_timestamp=False)
```

Comment:
I think in this case, the behavior was actually intended (see issue #1276): If a page was outdated and is then imported unchanged, this should update the status to "up to date".
However, with our recent changes of always creating new versions rather than modifying existing versions, I guess it would be more consistent to also create a new version here than to update the last updated date.
We can just do so by combining the last two conditions (so `elif not has_changed:` and `else` into one case and always delete the links and execute `page_translation.save()` and only adapt the info message depending on whether the page was actually changed or not.
Even though this is not strictly speaking part of this issue, could you add a commit with this change as well? Thanks in advance!